std/traits.d

```diff
@@ -3459,6 +3466,31 @@ unittest    // duplicated values
     static assert([ EnumMembers!A ] == [ A.a, A.b, A.c, A.d, A.e ]);
 }
 
+unittest // huge enums
```

Comment:
Could you comment with `\\ bugzilla 14561` instead ? It seems that this is widely adopted.
If someone read the code, with  `\\ huge enums`, it's not clear that it used to be a bug.